priam/src/test/groovy/com/netflix/priam/cluser/management/TestCompaction.groovy

```diff
@@ -0,0 +1,81 @@
+package com.netflix.priam.cluser.management
```

Comment:
Can you include tests to verify the concurrent executions?

Reply:
Done.